src/BigPipeTrait.php

```diff
@@ -47,4 +69,25 @@ public function bigPipeBeforeScenarioInit(BeforeScenarioScope $scope) {
     }
   }
 
+  /**
+   * Prepares Big Pipe NO JS cookie if needed.
+   *
+   * @BeforeStep
+   */
+  public function bigPipeBeforeStep(BeforeStepScope $scope) {
+    if ($this->skipBigPipeBeforeStep) {
+      return;
+    }
+    try {
```

Comment:
@richardgaunt 
could you please update this to check if the session exists.
if it does - try getting the cookie.

The exception would be thrown only in case if the session does not exist or the driver does not support it.

Reply:
I attempted to split out this into it's own try / catch but if you look at the step, we only try and get cookie if its a no javascript step and only then do we get the cookie - I couldn't see the point for splitting out.